orb/core/collection.py

```diff
@@ -250,11 +250,11 @@ def count(self, **context):
 
                 try:
                     with ReadLocker(self.__cacheLock):
-                        count = self.__preload['count'][context]
+                        count = self.__preload['count'][context] or 0
```

Comment:
aggregated data from the database when the records are empty will return as None.  This will just setup the proper defaults if we have already expanded & loaded data from the database, but it is blank.